content/css/jenkins.css

```diff
@@ -1449,3 +1450,4 @@ p.details {
 .supporters a {
   font-weight: bold;
 }
+
```

Comment:
I think I already wrote that in one of your previous PR @arshdkhn1 : review your changes locally, before and after committing, and try to always resist committing whitespace changes and so on. And before pushing. And then, think about how to make commits cleaner if need be, reorder them, reword them, split them, merge them. And then finally, push and possibly file a PR.

This will always lower your chances to get your changes merged in. Since 1) your change/commits won't be cleanly containing only what it's about, and 2) reviewers will get distracted by things unrelated to the goal of the PR.